soh/src/code/sys_matrix.c

```diff
@@ -552,15 +552,15 @@ Mtx* Matrix_MtxFToMtx(MtxF* src, Mtx* dest) {
     return dest;
 }
 
-Mtx* Matrix_ToMtx(Mtx* dest, char* file, s32 line) {
-    FrameInterpolation_RecordMatrixToMtx(dest, file, line);
-    guMtxF2L(Matrix_CheckFloats(sCurrentMatrix, file, line), dest);
+Mtx* Matrix_ToMtx(Mtx* dest) {
+    FrameInterpolation_RecordMatrixToMtx(dest, __FILE__, __LINE__);
```

Comment:
For example, you'd just continue to pass in the file and line function arguments to this frame interpolation function here.